app/src/main/java/eu/kanade/tachiyomi/ui/manga/track/TrackSearchDialog.kt

```diff
@@ -64,7 +62,9 @@ class TrackSearchDialog : DialogController {
             customView(viewRes = R.layout.track_search_dialog, scrollable = false)
             negativeButton(android.R.string.cancel)
             positiveButton(R.string.clear) { onPositiveButtonClick() }
-            setActionButtonEnabled(WhichButton.POSITIVE, wasPreviouslyTracked)
+            if (wasPreviouslyTracked) {
```

Comment:
This hides the button when it was not previous tracked.

setActionButtonEnabled only changes the color of the button so it was blue when previous tracked but still showed when not previously tracked